.nvmrc

```diff
@@ -0,0 +1 @@
+4.6.0
```

Comment:
I just wanted to go with the latest version that is under [TLS](https://github.com/nodejs/LTS#lts-schedule).

4.6.0 includes the latest security patches, but if you prefer v4.0 or if I am missing a major perk of going with 4.0, please remind me. :)